src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,273 @@
+
+### Tools
+
+Certain tools are essential for effectively using Daytona. Below is a list of tools that will facilitate interaction and development within the Daytona environment:
+
+| **Tool**                          | **Description**                                                                                                                                                    |
+|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Command-Line Interface (CLI)      | Essential for executing Daytona commands and interacting with the system.                                                                                          |
+| Text Editor/IDE                   | Recommended for editing configuration files and working with code within Daytona Workspaces.                                                                       |
+| SSH Client                        | Required for accessing remote servers, especially if you are deploying Workspaces on a remote Docker installation or accessing remote Git repositories.            |
+
+## Installing Daytona
+
+To install Daytona on your system, follow the instructions provided in the [Installation](/installation/installation) guide. The guide includes detailed steps for installing Daytona on various operating systems.
+
+Ensure that all necessary system requirements, dependencies, and tools listed above are installed and properly configured on your system. The installation process may vary depending on your operating system and specific environment, so it is essential to refer to the guide for comprehensive instructions tailored to your setup.
+
+Once you have completed the installation, you will be ready to start using Daytona to manage and develop your development environments. If you encounter any issues during installation, consult by reaching out to Daytona's [Slack community](https://go.daytona.io/slack) for assistance.
+
+<DocumentList>
+<DocumentListItem
+    title="Install Daytona"
+    subtitle="Learn how to install Daytona on Linux, macOS, or Windows."
+    href="installation/installation"
+    expanded
+  />
+</DocumentList>
+
+## Adding a Git Provider
+
+To interact with Git repositories and manage version control operations, you can add a [Git Provider](/configuration/git-providers) to Daytona. This step is essential for cloning repositories, pushing changes, and managing your codebase effectively.
+
+Daytona supports a wide range of Git Providers, including popular platforms like [GitHub](/configuration/git-providers#github), [GitLab](/configuration/git-providers#gitlab), and [Bitbucket](/configuration/git-providers#bitbucket), as well as self-hosted solutions. Integrating a Git Provider with Daytona allows you to manage your repositories directly within the platform, eliminating the need for repeated authentication.
```

Comment:
`eliminating the need for repeated authentication.`

I suggest we remove this part because if you attempt to create a project from a private repo, it will just fail. It won't ask for auth